# Clears cartesian pose buffer
---
bool success
string msg
